devops/ansible/site.yml

```diff
@@ -4,106 +4,179 @@
   vars:
     required_ansible_version: 2.5.1
   tasks:
-    - fail: msg="Playbook requires ansible >= {{required_ansible_version}}"
+    - fail: msg='Playbook requires ansible >= {{ required_ansible_version }}'
       when: ansible_version.string is version_compare(required_ansible_version, '<')
 
     - name: Set some facts
       set_fact:
-        girder_host: "{{girder_host|default('localhost')}}"
-        girder_port: "{{girder_port|default(8080)}}"
-        girder_scheme: "http"
+        girder_host: '{{ girder_host|default("localhost") }}'
+        girder_port: '{{ girder_port|default(8080) }}'
+        girder_scheme: 'http'
 
     - name: Create Girder eSimMon admin user
       girder:
-        host: "{{girder_host}}"
-        port: "{{girder_port}}"
-        scheme: "{{girder_scheme}}"
+        host: '{{ girder_host }}'
+        port: '{{ girder_port }}'
+        scheme: '{{ girder_scheme }}'
         user:
-          firstName: "esimmonadmin"
-          lastName: "esimmonadmin"
-          login: "esimmonadmin"
-          password: "{{ admin_password }}"
-          email: "esimmonadmin@kitware.com"
+          firstName: 'esimmon'
+          lastName: 'esimmon'
+          login: 'esimmon'
+          password: '{{ admin_password }}'
+          email: 'brianna.n.major@gmail.com'
```

Comment:
Not sure you want to include this? You might get alot of spam ( may be too late! )